.circleci/config.yml

```diff
@@ -31,9 +31,7 @@ jobs:
             mkdir -p /tmp/hub
             cd /tmp/hub
 
-            curl -L -o hub.tgz https://github.com/github/hub/releases/download/v2.10.0/hub-linux-amd64-2.10.0.tgz
-            tar -xzvf hub.tgz --strip 1
-            sudo mv ./bin/hub /usr/local/bin
+            apt-get install -y hub
```

Comment:
is hub in apt the latest version? i know apt always provide some really old version of packages :joy: